collections/_pages/features.html

```diff
@@ -50,7 +50,7 @@
       - text:
         highlight:
     call_to_action:
-      link: /#getting-started
+      link: https://docs.meshery.io/functionality/lifecycle-management#configuration-management
```

Comment:
to link configuration feature, I'm not sure about the proper link. plz suggest here.

Reply:
@afzal442, yikes! You're right. Let's get an issue opened in the Meshery Docs to create a new page for configuration management.

Reply:
Do you mind opening this docs issue, @afzal442?

Reply:
Ohk sure. Do you mean the objective will be to transfer the content of configuration-management to another page or just create a new page describing config management?